src/js/labels/labels.ts

```diff
@@ -1,5 +1,14 @@
 export type LabelOptions = { seekOffset?: number; playbackRate?: number };
 
+// Setting a code explicitely to null makes the error not show up in the UI.
+export const errors = {
```

Comment:
question: is there a benefit to not implementing this as a function that accepts e.g. an "error-shaped object" or similar and yields a (potentially empty) string? That would be consistent with other label code below and should simplify the error dialog impl

Reply:
I'm now seeing we're also relying on this string to be a mini "domain specific language"/syntax in the error dialog parsing code. My S1 is to break these out into two functions - one for a default title by error shape (code and/or message), another for a default verbose message by error shape (code and/or message).

Reply:
Still not a fan of this being an object with the assumption of potentially mutating it, but I won't press here.